src/react/2-intro-to-jsx.md

```diff
@@ -34,7 +34,7 @@ React.createElement(
 );
 ```
 
-The code above produces the following HTML:
+The code above produces a something similar to the following HTML:
```

Comment:
I believe it's better to err on the side of specificity and avoid writing "a something".

Reply:
Agreed. I'm gonna get rid of "a something" and just write "something" here. My only hesitation with writing the first sentence, is that React.createElement doesn't actually return HTML, it returns VDOM. Maybe there is a better way to phrase this than "The code above produces something similar to the following HTML", but I'm not sure what it is. Hit me up with any ideas.

Reply:
My intent with it was that it _eventually_ produces HTML once its rendered to screen. But I absolutely see what you mean. Perhaps something along the lines of "The code above will produce the following HTML once rendered"?

Reply:
In any case -- reading "something similar to" without a statement of what it actually amounts to sounds like its leaving out information.

Reply:
Love it. Added that as a sentence.